src/app/test-record/test-delete-reason-modal/test-delete-reason-modal.component.scss

```diff
@@ -0,0 +1,26 @@
+.enter-reason-for-cha {
+  color: #0b0c0c;
+  font-size: 36px;
+  font-weight: 700;
+  line-height: 43px;
+  text-align: left;
+  width: fit-content;
+}
+textarea {
```

Comment:
remove that, this is not needed and will override all the textarea inputs.
Let's not use `!important`, this is forbidden.